specs/gluster_object_detail_views.adoc

```diff
@@ -0,0 +1,198 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of volume and host associated with a gluster cluster discovered by Tendrl
+
+== Problem description
+
+Display list of volume and host associated with a gluster cluster present in Tendrl. 
+
+*Host List*
+
+The details shown for each host are -
+
+* Status
+* Name
+* Role
+* Alerts
+
+The host list should have functionality for 
+
+* filtering by name, role and status.
```

Comment:
If the same host list view would be used for other storage systems such as Ceph, then the `role` column is essential. Even in gluster's case, only a specific node would have the role of provisioner (cluster specific and/or global), which needs to be shown.

The UI would need to add specific logic to be able to display the provisioner roles. I figure this logic would be to look for the tags `provisioner/gluster` and `provisioner/<cluster_id>`. These roles would map to the terms `Global Gluster Provisioner` and `Provisioner` respectively.

BTW, should the role `gluster/server` be renamed to `gluster/peer` on the backend? I believe that's the official gluster terminology. When changed, this would also enable the role for every node in gluster to be displayed as `Peer` in the UI.

@gnehapk @shtripat @nnDarshan @r0h4n